### DISCUSSION TITLE
Llama parse chunking text by itself?

### DISCUSSION THREAD

**Discussion by ansar-ayan** [run-llama/llama_cloud_services#320] [Q&A]
Hey guys, just wanted to ask if any update has been made to llama parse to chunk the incoming pdf file itself. If yes. is there a way i can stop that? 

because i remember text not being chunked before

**Comment by aaqib-ahmed-nazir** [comment]
Yep, Llama Parse now chunks PDFs automatically. There's no official way to stop it yet. Possible workarounds include manual pre-processing or modifying the code (risky).

**Reply by ansar-ayan** [reply]
` parser = LlamaParse(
            api_key="API-KEY",
            result_type="markdown",
            verbose=False,
            num_workers=8,
            split_by_page=0,
        )`

This is a possible workaround I found while testing. You need to set split_by_page = 0.
